app/lib/actions/index.js

```diff
@@ -1,7 +1,5 @@
-import { requestSession } from './sessions';
+import rpc from '../rpc';
 
 export function init () {
-  return (dispatch) => {
-    dispatch(requestSession());
-  };
+  rpc.emit('init');
```

Comment:
so, before we were delaying `init` until we got a session going. The reason for this is that we don't want to `show` the window until the user has something to do pretty much (i.e: a prompt). Why was this changed?